cocos/renderer/ccShaders.cpp

```diff
@@ -92,4 +92,7 @@ NS_CC_BEGIN
 #include "renderer/ccShader_CameraClear.vert"
 #include "renderer/ccShader_CameraClear.frag"
 
+// x-studio365 spec
+#include "renderer/ccShader_ETC1AS_PositionTextureColor.frag.ipp"
+#include "renderer/ccShader_ETC1AS_PositionTextureGray.frag.ipp"
```

Comment:
It's from c++ boost library, It's mean the file contains impl-code which included in a cpp file.